src/main/java/com/bobocode/svydovets/annotation/Component.java

```diff
@@ -5,9 +5,16 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+import com.bobocode.svydovets.beans.definition.BeanDefinition;
+
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
 
+    /**
+     * Specify the name of {@link BeanDefinition} associated with annotated class.
```

Comment:
Does a user need to know about BeanDefinition? I think we should hide this detail from the user.